utilities/provider_tallies/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "provider_tallies"
+version = "0.0.0"
+description = "Provider tallies scripts for Openverse"
+authors = [
+  {name = "Openverse Contributors", email = "openverse@wordpress.org"},
+]
+dependencies = [
+    "redis ~= 5.0",
+    "pandas ~= 2.2",
+    "click ~= 8.1",
+    "matplotlib ~= 3.9",
+]
+requires-python = "==3.12.*"
+readme = "README.md"
+license = {text = "MIT"}
+
```

Comment:
I think we can remove these lines because

- there isn't a README in the folder
- the license is unspecified (or maybe the outer `LICENSE` applies idk?)
- extra blank line

Reply:
This folder actually _does_ have a README.
But in others, `pdm init` adds a lot of `__init__.py` files, as well as a blank README. So, I'll double check that there's no blank README left